pkg/sfu/buffer/rtpstats.go

```diff
@@ -174,6 +174,82 @@ func NewRTPStats(params RTPStatsParams) *RTPStats {
 	}
 }
 
+func (r *RTPStats) Seed(from *RTPStats) {
```

Comment:
This means logging of RTPStats will be cumulative, i. e. before migration RTPStats will show some numbers. Then this RTPStats carries on from where the previous instance left off. So, when this session ends, logging will show since start of the track. Have to take that into account while debugging.

Reply:
Wondering if there is a way to avoid forget copy some fields, there are so much fields here :)

Reply:
I know @cnderrauber . It is painful :-) Will think about better ways to do these.